ofrak_core/ofrak/gui/server.py

```diff
@@ -916,6 +979,43 @@ async def get_tags_and_num_components(self, request: Request):
                 all_resource_tags_l.remove(resource_tag)
         return json_response(all_resource_tags_l)
 
+    @exceptions_to_http(SerializedError)
+    async def search_data(self, request: Request) -> Response:
+        resource: Resource = await self._get_resource_for_request(request)
+        body = await request.json()
+        mode = body.get("type")
```

Comment:
Anywhere in the back end that it gets `type` from `options` needs to be changed to `searchType`, or `searchType` needs to be renamed to `type` in the front end. Either way, the `...options` change made it so the JSON passed to the back end uses all of the exact names from the front end. 

```suggestion
        mode = body.get("searchType")
```